### PR TITLE
Added function to print logging info for active window

### PR DESCRIPTION
**Pull request by derekthecool**
Workspacer is already keyboard oriented, this should be a nice addition
for anyone who wants to use just keyboard.

I found myself wanting this feature while setting up routes, and getting annoyed by using the mouse.
Also getting annoyed by having to scroll up through the entire list of windows.

### CODE REVIEW

File: src/workspacer.Native/Native/WindowsManager.cs [hunk 1]
```diff
@@ -89,6 +89,19 @@ public void DumpWindowDebugOutput()
...
 
+        public void DumpWindowDebugOutputForFocusedWindow()
+        {
+            var output = "";
+            var focusedWindow = Windows.Where(win => win.IsFocused)
+                .Select(win => win).FirstOrDefault();
+
+            if (focusedWindow != null)
+            {
+                output += GenerateWindowDebugOutput(focusedWindow) + "\r\n\r\n";
```
Comment (josteink) [inline src/workspacer.Native/Native/WindowsManager.cs:100]: Output has previously only been assigned a blank value, and we don't need to concatenate to it. `output = ` alone will be fine.

Also: Any reason we need to append those newlines at the end? :smile:

**Comment by josteink** [comment]
Hey there and thanks for the patch!

Changes looks mostly good to me, but I see there are a few small things which can be cleaned up before we merge :)

File: src/workspacer.Native/Native/WindowsManager.cs (hunk 1 above)
Comment (derekthecool) [reply src/workspacer.Native/Native/WindowsManager.cs]: Done

**Review by josteink — APPROVED** [review]
LGTM!

**Pull request merged**